Tasks/FtpUploadV2/Tests/package-lock.json

```diff
@@ -1,8 +1,24 @@
 {
   "name": "ftp-upload-tests",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
@anatolybolshakov maybe we should revert this change to the `lockfileVersion`

Reply:
@DaniilShmelev as I know v2 version is backward compatible to previous versions of npm - so there should not be issues